website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -390,6 +397,7 @@ Properties without a default value are required.
 | `data.attributes.cost-estimation-enabled`                                 | boolean | true      | Whether or not the cost estimation feature is enabled for all workspaces in the organization. Defaults to true. In a Terraform Cloud organization that does not have Teams & Governance features, this value is always false and cannot be changed. In Terraform Enterprise, Cost Estimation must also be enabled in Site Administration.                                          |
 | `data.attributes.send-passing-statuses-for-untriggered-speculative-plans` | boolean | false     | Whether or not to send VCS status updates for untriggered speculative plans. This can be useful if large numbers of untriggered workspaces are exhausting request limits for connected version control service providers like GitHub. Defaults to false. In Terraform Enterprise, this setting is always false and cannot be changed but is also available in Site Administration. |
 | `data.attributes.owners-team-saml-role-id`                                | string  | (nothing) | **Optional.** **SAML only** The name of the ["owners" team](/enterprise/user-management/saml/team-membership#managing-membership-of-the-owners-team)                                                                                                                                                                                                                               |
+| `data.attributes.assessments-enforced`                                     | boolean | (false)   | Whether or not to compel health assessments for all workspaces capable of being assessed. When true, health assessments will occur on all compatible workspaces, regardless of the value of the workspace setting `assessments-enabled`. When false, health assessments will only occur for workspaces that opt in by setting `assessments-enabled: true`.
```

Comment:
We don't want to use "will" --> Our style guide prohibits future-facing words to describe current functionality. Also --> this isn't part of TFE, so I was thinking we should say that here?

```suggestion
| `data.attributes.assessments-enforced`                                     | boolean | (false)   | Whether or not to compel health assessments for all eligible workspaces. When true, health assessments occur on all compatible workspaces, regardless of the value of the workspace setting `assessments-enabled`. When false, health assessments only occur for workspaces that opt in by setting `assessments-enabled: true`. Not available in Terraform Enterprise.
```